src/io/flutter/logging/FlutterLogFilterPanel.java

```diff
@@ -97,8 +99,7 @@ protected void customizeCellRenderer(@NotNull JList<? extends FlutterLog.Level>
                                            int index,
                                            boolean selected,
                                            boolean hasFocus) {
-        final String label = index == -1 && value == FlutterLog.Level.NONE ? "" : value.name().toLowerCase();
-        append(label);
+        append(value.toDisplayString());
```

Comment:
In previous code -> `Level.NONE` will display nothing (empty string).
But for new code -> I think it will display `none`.

Reply:
Yup, this is an explicit change. I thought eliding none would be useful, but after using the view for a while I'm not sure. Definitely odd to not show it if the default setting is `config`.